server/protocols/jmap-rfc-8621/src/main/scala/org/apache/james/jmap/method/EmailGetMethod.scala

```diff
@@ -153,6 +153,7 @@ class EmailGetMethod @Inject() (serializer: Serializer,
       .groupBy(_.getMessageId)
       .flatMap(groupedFlux => groupedFlux.collectSeq().map(results => (groupedFlux.key(), results)))
       .map(Email.from)
+      .map(_.get)
```

Comment:
You should not call 'get' on a Try.

replace the last two 'map' by :
` .flatMap(email => SMono.fromTry(Email.from(email)))`